src/Logic/__test__/operate.test.js

```diff
@@ -0,0 +1,19 @@
+import operate from '../operate';
+
+describe('operations', () => {
+  test('it adds numbers', () => {
+    expect(operate(2, 2, '+')).toBe('4');
+  });
+  test('it subtracts numbers', () => {
+    expect(operate(2, 2, '-')).toBe('0');
+  });
+  test('it multiplies numbers', () => {
+    expect(operate(2, 2, 'x')).toBe('4');
+  });
+  test('it divides numbers', () => {
+    expect(operate(2, 2, '÷')).toBe('1');
+  });
```

Comment:
- [ ] [OPTIONAL] - Seeing tests in projects makes everyone happy :smiley: . What if you now stretch it to test scenarios such as where there are negative values in the operations?